template_config.yaml

```diff
@@ -0,0 +1,23 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+project:
+    name: "project-slug"
+    modalities: ["DX", "CR"] # DICOM dataset modalities to retain
+
+tag_operation_files: ["base-tag-operations.yaml"] # DICOM tag anonymisation operations, can specify multiple files
+
+destination:
+    dicom: "ftps" # alternatives: "none", "dicomweb", "azure"
```

Comment:
Just adding here as well, I don't think we should be offering options which aren't implemented